sqlmesh/core/model/definition.py

```diff
@@ -101,9 +100,6 @@ class _Model(ModelMeta, frozen=True):
             The start date can be a static datetime or a relative datetime like "1 year ago"
         end: The date that the model will be backfilled up until. Follows the same syntax as 'start',
             should be omitted if there is no end date.
-        batch_size: The maximum number of incremental intervals that can be run per backfill job. If this is None,
```

Comment:
why was the doc removed? duplicate?

Reply:
`batch_size` hasn't been a top-level model attribute for a long time now.